binary_transparency/firmware/internal/client/follower.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
```

Comment:
should the name be more appropriate as logfollower or something similar?

Reply:
I'm happy with the current name as its a) short, and b) I'd be quite happy to stick and other code for followers (e.g. MapFollower) in this file if needed. We can always rename it later if we want. Go generally prefers shorter naming.